elide-model-config/src/main/resources/elideTableSchema.json

```diff
@@ -33,7 +39,7 @@
             "required": [
                 "name",
                 "to",
-                "type",
+                "kind",
```

Comment:
Let's make this optional and default to toOne.

Reply:
done.